descriptors/ModuleDescriptor-template.json

```diff
@@ -1954,18 +1954,19 @@
       "displayName": "module permissions for one op",
       "description": "to reduce X-Okapi-Token size",
       "subPermissions": [
-        "inventory-storage.items.item.get",
         "inventory-storage.items.collection.get",
         "inventory-storage.holdings.collection.get",
-        "inventory-storage.holdings.item.get",
         "inventory-storage.instances.collection.get",
-        "inventory-storage.instances.item.get",
+        "inventory-storage.material-types.collection.get",
+        "inventory-storage.loan-types.collection.get",
         "inventory-storage.service-points.collection.get",
-        "inventory-storage.service-points.item.get",
-        "inventory-storage.locations.item.get",
         "inventory-storage.locations.collection.get",
-        "circulation-storage.requests.item.get",
-        "circulation-storage.requests.collection.get"
+        "inventory-storage.location-units.libraries.collection.get",
+        "inventory-storage.location-units.campuses.collection.get",
+        "inventory-storage.location-units.institutions.collection.get",
+        "circulation-storage.requests.collection.get",
+        "users.collection.get",
+        "addresstypes.collection.get"
```

Comment:
Why have these permissions been reordered? What was the thought process in re-ordering them?

The collection and item variants are typically both included as to avoid minor changes to how the records are fetched causing permissions failures.

Reply:
I realized that the initial list was inadequate considering the latest changes, so I completely removed it and reassembled with permissions we actually need to make this functionality work. I will add the `*.item.*` permissions.

Reply:
Thank you

Reply:
done